document HPA's implicit deactivation (#24825)
kubernetes/website: content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

    selectPolicy: Disabled
```

> ## Support for implicit maintenance-mode deactivation

Reviewer: ```suggestion
## Implicit maintenance-mode deactivation
```

> Starting from v1.4, you can implicitly deactivate the HPA for a target without the

Reviewer: ```suggestion
You can implicitly deactivate the HPA for a target without the
```
(we assume readers are using a supported version of Kubernetes)

need to change the HPA configuration itself. If the target's desired replica count

> is set to 0, and the HPA's minimum replica count is greater than 0, the HPA will
> cease to adjust the target (and will set the `ScalingActive` Condition on itself

Reviewer: nit:
```suggestion
is set to 0, and the HPA's minimum replica count is greater than 0, the HPA 
stops adjusting the target (and sets the `ScalingActive` Condition on itself
```
